skycoin/skywire
# Use page state management

Did you run `make format && make check`?
The go code was not changed. `npm run build` wes used. `npm run lint` could have problems for the update to Angular 16.

 Changes:	
- The last commit makes the whole app use page state management, as it is more usable with the new verisons of Angular. This makes the app remember the system state while navigating, which means remembering the scroll position, avoid reloading previously loaded data from the server and other usability improvements. This should be more noticeable on mobile devices.

This is similar a previous change made to the explorer, but with some changes, because the router strategy used by the Hypervisor UI in the router config is different.

How to test this PR:
After recompiling the front-end, when navigating with the browser buttons the app should now remember the scroll position and some parts of the app should load a bit faster.
